linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -131,9 +134,9 @@ public static void main(String[] arguments)
         Model model = modelBuildingResult.getEffectiveModel();
         artifactToVersion.put(model.getGroupId() + ":" + model.getArtifactId(), model.getVersion());
       } catch (ModelBuildingException ex) {
-        // When there's a pom.xml file that is not (indirectly) referenced by the root pom.xml,
-        // Maven may fail to build the model. Such pom.xml can be ignored.
-        System.out.println("Ignoring bad model: " + path);
+        // Maven may fail to build pom.xml files found in irrelevant directories, such as "target"
```

Comment:
should we deliberately and explicitly skip those files?

Reply:
I'd keep the logic simple until we need it.